features/steps/run_steps.py

```diff
@@ -75,8 +75,9 @@ def list_kedro_pipelines(context):
 @when("I lint the project")
 def lint_project(context):
     context.result = subprocess.run(
-        [context.kedro, "lint", "--check-only"], cwd=context.root_project_dir
+        [context.python, "-m", "ruff", "check", "src"], cwd=context.root_project_dir
     )
+    print(context.result.stdout)
```

Comment:
Is this added on purpose?

Reply:
Oh oops good catch! I used that for debugging.